pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -79,6 +79,7 @@ type CertificateSpec struct {
 	// CommonName is a common name to be used on the Certificate.
 	// The CommonName should have a length of 64 characters or fewer to avoid
 	// generating invalid CSRs.
+	// This value is ignored by TLS clients when any subject alt name is set.
```

Comment:
Could you clarify here that this is x509 behaviour and not something specific to cert-manager by linking to https://tools.ietf.org/html/rfc6125#section-6.4.4 ? 😄 

This change also needs to be reflected in the v1alpha3 docs, and you'll need to run `./hack/update-crds.sh` afterwards too!